lib/ovirt/inventory.rb

```diff
@@ -130,7 +130,7 @@ def standard_collection(uri_suffix, element_name = nil, paginate = false, sort_b
     SECONDARY_ITEMS = {
       # Key          RHEVM API methods
       :datacenter => [:storagedomains],
-      :host       => [:statistics, :nics], # :cdroms, tags
+      :host       => [:statistics, :host_nics], # :cdroms, tags
```

Comment:
Won't this break inventory?  I don't see it listed in the docs anywhere.  https://access.redhat.com/documentation/en-US/Red_Hat_Enterprise_Virtualization/3.4/html/Technical_Guide/chap-Hosts.html  https://access.redhat.com/documentation/en-US/Red_Hat_Enterprise_Virtualization/3.6/html/REST_API_Guide/chap-Hosts.html